crates/model/src/order.rs

```diff
@@ -73,6 +75,9 @@ impl Order {
             },
             signature: order.signature.clone(),
             data: order.data,
+            // Currenlty, we only support pre_interactions created
+            // via the on-chain order parsing
+            pre_interactions: Vec::new(),
```

Comment:
In order to be "forward compatible", should we introduce:
```rust
pub struct Interactions {
    pub pre: Vec<InteractionData>,
    // And in the future:
    //pub intra: Vec<InteractionData>,
    //pub post: Vec<InteractionData>,
}
```

Reply:
Hm... But I can not imagine any uses-cases for it. Should I still go for it?

Reply:
For `post` interactions, one use case could be for bridging funds to other networks.

Reply:
I'm not super attached to this idea, and I think its OK to leave it out for now, I just thought it would be good to group all order-specific interactions into its own data-type.

Reply:
API-breaking changes scare me! I will do it. 
Good point. Convinced.